tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -371,8 +369,9 @@ Status StorageManagerCanonical::async_push_query(Query* query) {
       [this, query]() {
         // Process query.
         Status st = query_submit(query);
-        if (!st.ok())
-          logger_->status_no_return_value(st);
+        if (!st.ok()) {
+          resources_.logger()->status_no_return_value(st);
+        }
```

Comment:
This really should be improved, but it's out of scope for the present PR, for which this change is minimally acceptable.